src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/localize/templatestrings.en.json

```diff
@@ -3,7 +3,9 @@
   "name": "API Controller",
   "description": "API Controller with or without read/write actions",
   "symbols/namespace/description": "namespace for the generated code",
+  "_symbols/namespace/description.comment": "Should this start w/ 'Namespace'?",
   "symbols/actions/description": "create controller with read/write actions",
-  "symbols/actions/displayName": "Add ReadWrite Actions",
-  "postActions/openInEditor/description": "Opens ValueController.cs in the editor"
-}
+  "_symbols/actions/description.comment": "Should this start w/ 'Create'?",
+  "symbols/actions/displayName": "Add Read/Write Actions",
+  "postActions/openInEditor/description": "Opens the created controller in the editor"
```

Comment:
To see exactly which strings I changes (as opposed to strings that were just newly copied from the template.json files automatically), look at changes in the template.json files. For example, I changed the `displayName` resource but not the post-action `description`.